application/common/mail/mfa-disabled.html.php

```diff
@@ -26,4 +26,29 @@
 ?>
 <p>Dear <?= yHtml::encode($displayName) ?>,</p>
 
-<h2>TODO: Compose contents of this email message</h2>
\ No newline at end of file
+<p>
+    2-Step Verification has been disabled on your <?= yHtml::encode($idpDisplayName) ?> Identity account. If this was not intentional
+    follow the instructions below to setup 2-Step Verification on your account.
+</p>
+<p>
+    If you did not do this it could be a sign someone else has compromised your account.
+    Please contact <?= yHtml::encode($supportName) ?> at <?= yHtml::encode($supportEmail) ?> as soon as possible to report the incident.
+</p>
+
+<p><b>Instructions to set up 2-Step Verification:</b></p>
+<ol>
+    <li>Go to <?= yHtml::encode($passwordProfileUrl) ?></li>
+    <li>Under 2-Step Verification, set up the options that suit you best (Security Key, Smartphone App, and/or
+    Printable Codes)</li>
+    <li>Log out and log in again to see how it works and to have it remember your computer for 30 days. Note that
+        logging out will undo the "Remember this computer" setting.</li>
```

Comment:
Logging out will no longer undo the "Remember" setting, so we probably want to remove this sentence.

Reply:
Done